enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+---
+title: sdk-fbc-run-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+approvers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+creation-date: 2022-04-20
+last-updated: 2022-04-20
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Add FBC support to run bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+- Do we need to build a File-Based Catalog to run this command?
+
+  - In the case where an index image isn't provided, we need to build a File-Based Catalog on the fly when `run bundle` command is invoked.
+
+- Do we need to make our own image to use as a default one for `run bundle` and `run bundle-upgrade`?
+
+  - Currently, we are using `quay.io/operator-framework/opm:latest` as the default index image. This image should suffice as the default image for both index formats (FBC and SQLite images).
+
+- While initializing the FBC, should the channel received from the bundle annotations be used to initialize the default channel name in the `opm init` command?
+
+  - We should be able to handle bundles that don't have channel metadata on the annotations.
+
+- If we don’t get channel information from the bundle, can we add it to a default channel or to the channel HEAD?
+
+  - For FBC, channel membership and package-level metadata embedded in bundles is ignored. So we should design FBC tooling to ignore that as well. The channel name can either be hardcoded (e.g. channel name = `develop`) or it can be derived. Hardcoding the channel name may be better as deriving it to a name that contains the package name seems to be redundant as the channel is already a member of the package.
+
```

Comment:
Why does someone author want to use a run bundle informing the index?
Is that to check if is possible to add the new bundle version and etc to the index? 
If yes, are we properly addressing this need by ignoring the channel? 
c/c @joelanford